main.go

```diff
@@ -105,7 +106,11 @@ func main()  {
 	Addr := fmt.Sprintf("%s%s%d", *host, ":", *port)
 
 	for i := 0; i < *concurrency; i++ {
-		go ProcessOutgoingConnection(Addr, tlsConfigs[i], sleep, message)
+		if *parallel == true {
```

Comment:
Why ` == true`?

Reply:
What do you mean? How do you compare boolean flags?

Reply:
I don't **compare** them. I **check** them. Just remove the mentioned characters from the code and you'll see what I mean.